src/app/lib/config/services/thai.js

```diff
@@ -29,7 +29,7 @@ export const service = {
     locale: `th-TH`,
     datetimeLocale: 'th',
     service: 'thai',
-    serviceName: 'News ไทย',
+    serviceName: 'Thai',
```

Comment:
Looking at the codebase, I don't believe this `serviceName` value is used any longer. (We should remove it). Is there a reason why you're updating it here? Where did you want this value to be used?

Reply:
In this case, I’m updating them to be consistent between the services, based on what we set up in Arabic when we launched Articles there.

Reply:
Okay, I see, thanks.